AdminWebsite/AdminWebsite/ClientApp/src/app/services/clients/api-client-base.ts

```diff
@@ -0,0 +1,6 @@
+export class ApiClientBase {
+    protected transformOptions(options: any) {
+        options.headers = options.headers.append('Cache-Control', 'no-store');
+        return Promise.resolve(options);
+    }
+}
```

Comment:
yes, correct. You specify an ApiClientBase class for nswag to use, and then the tool generates the api-client. It puts the ApiClientBase into the api-client I believe so that the functions can reference it.